### PR TITLE
Adding content/pt/docs/reference/index

### PR DESCRIPTION
**Pull request by femrtnz**
Hey guys

Please review this PR when you have time :)

### CODE REVIEW

File: content/pt/docs/reference/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+* [kubectl](/docs/reference/kubectl/overview/) - Ferramenta CLI principal para executar comandos e gerenciar clusters do Kubernetes.
+    * [JSONPath](/docs/reference/kubectl/jsonpath/) - Guia de sintaxe para usar [Expressões JSONPath](http://goessner.net/articles/JsonPath/) com o kubectl.
+* [kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm/) - Ferramenta CLI para provisionar facilmente um cluster Kubernetes seguro.
+
+## Referência de configuração
+
+* [kubelet](/docs/reference/command-line-tools-reference/kubelet/) - O principal * agente do nó * que é executado em cada nó. O kubelet usa um conjunto de PodSpecs e garante que os contêineres descritos estejam funcionando e saudáveis.
+* [kube-apiserver](/docs/reference/command-line-tools-reference/kube-apiserver/) -API REST que valida e configura dados para objetos de API, como pods, serviços, controladores de replicação.
+* [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/) - Daemon que incorpora os loops de controle principais enviados com o Kubernetes.
```
Comment (devlware) [inline content/pt/docs/reference/_index.md]:
```suggestion
* [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/) - Daemon que incorpora os princiapis loops de controle enviados com o Kubernetes.
```

File: content/pt/docs/reference/_index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) - É possível fazer o encaminhamento de fluxo TCP/UDP  de forma simples ou utilizando o algoritimo de Round Robin encaminhando para em um conjunto de back-ends.
```
Comment (devlware) [inline content/pt/docs/reference/_index.md]:
```suggestion
* [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) - É possível fazer o encaminhamento de fluxo TCP/UDP de forma simples ou utilizando o algoritimo de Round Robin encaminhando através de um conjunto de back-ends.
```

File: content/pt/docs/reference/_index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* [Visão geral da API do Kubernetes](/docs/reference/using-api/api-overview/) - Visão geral da API para Kubernetes.
+* [Referência da API Kubernetes {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
+
+## Biblioteca de clientes da API
+
+Para chamar a API Kubernetes de uma linguagem de programação, você pode usar
+[bibliotecas de clientes](/docs/reference/using-api/client-libraries/). Bibliotecas oficialmente suportadas:
+
+- [Biblioteca em Go do cliente Kubernetes](https://github.com/kubernetes/client-go/)
+- [Biblioteca em Python do cliente Kubernetes](https://github.com/kubernetes-client/python)
```
[the remark this answers is not in the document]
Comment (femrtnz) [reply content/pt/docs/reference/_index.md]: Good point. Yes, make sense to me change it.

what you think of  `Biblioteca do cliente Kubernetes em X]`

[the remark this answers is not in the document]
Comment (devlware) [reply content/pt/docs/reference/_index.md]: Great.

**Comment by sftim** [comment]
/hold
pending v1.18 Kubernetes release

**Comment by VineethReddy02** [comment]
/hold cancel
v1.18 release is out.

**Pull request merged**